src/server/services/input-types.js

```diff
@@ -500,6 +500,14 @@ defineType({
 
         const MAX_DATE_LEN = 128;
         let dateLen = Math.min(MAX_DATE_LEN, input.length);
+        let deltaSepPos = 0;
```

Comment:
This doesn't look like it will work when there is no space before the delta. For example, `now-1d` doesn't work. Is this intentional?

Reply:
I wanted to just split on whitespace or +/-, but apparently the `Date` parser allows dashes for number separators, so for simplicity I just went the whitespace route. It doesn't help that the builtin `Date` accepts a ridiculous number of formats that we technically support but don't document for users.